### ISSUE TITLE
Suggestion: docker built-in webdav mounting

### ISSUE THREAD

**Issue by Lengso** [vh13294/crytomator-docker#2]
Suggestion: docker built-in webdav mounting

Use another program in the docker container to mount your own webdav and convert it to a directory

In this way, the host can mount two directories, one source directory and one encrypted directory.

No need to mount via webdav anymore

**Comment by vh13294** [comment]
Yeah, I've thought of that problem before but never got a chance to look into it.

In the mean time, you can do docker-compose with two different instances that include webdav-client.

[https://github.com/efrecon/docker-webdav-client](url)

**Comment by Lengso** [comment]
Your theory works. I'm trying to make docker-compose. I may need to modify docker-webdav-client. cryptomator-docker cannot set the username and password.
